bugbug/model.py

```diff
@@ -297,23 +336,36 @@ def train(self, importance_cutoff=0.15):
                     continue
 
                 y_test_filter.append(y_test[i])
-                y_pred_filter.append(argmax)
+                if is_multilabel:
+                    y_pred_filter.append(y_pred[i])
+                else:
+                    y_pred_filter.append(argmax)
 
-            y_pred_filter = self.le.inverse_transform(y_pred_filter)
+            if not is_multilabel:
+                y_pred_filter = self.le.inverse_transform(y_pred_filter)
 
             print(
                 f"\nConfidence threshold > {confidence_threshold} - {len(y_test_filter)} classified"
             )
+
             if len(y_test_filter) != 0:
-                labeled_confusion_matrix = get_labeled_confusion_matrix(
-                    y_test_filter, y_pred_filter, self.class_names
-                )
-                print(labeled_confusion_matrix)
-                print(
-                    classification_report_imbalanced(
-                        y_test_filter, y_pred_filter, labels=self.class_names
+                if is_multilabel:
+                    print_labeled_confusion_matrix(
```

Comment:
Here too, you can probably call print_labeled_confusion_matrix in both cases (np.asarray(y_test_filter) should work in both cases)